Makefile

```diff
@@ -42,6 +42,27 @@ endif
 build:
 	$(GO) build $(GO_LDFLAGS) $(BUILDV) $(BUILDTAGS) -o $(PKGNAME) $(BUILDFILES)
 
+PLATFORMS := darwin linux windows
+
+WIN_FLAGS = -ldflags "-X main.Version=$(VERSION) -linkmode=internal -s -w -extldflags '-static' -extld=x86_64-w64-mingw32-gcc"
+
+buildNatives:
+	@for platform in $(PLATFORMS); \
+	do \
+		arch="amd64"; \
+		output_name="$(PKGNAME)-$$platform"; \
+		if [ $$platform = windows ]; then \
+			echo $$output_name; \
+			echo "asdasddassad"; \
+			wintags='$(TAGS) nolzma'; \
```

Comment:
Never lzma support on windows?

Reply:
that was the go to solution, as the compiler complained about not being able to find the `lzma.h` and the exploratory nature